support oidc v2 (#43)
cyberark/conjur-api-python: README.md

### Configuring the client

#### Define Modules

> Authentication strategy supported by package are `authn`, `authn-ldap`, `authn-oidc` and based on authentication machanisam use the below module.

Reviewer: ```suggestion
Authentication strategies supported by this package are `authn`, `authn-ldap`, and `authn-oidc`. Based on authentication machanisam use the below module.
```

Author: code changed

> * Authn authentication (Supported on Conjur OSS,Enterprise,Cloud)

Reviewer: ```suggestion
* Authn authentication (Supported on Conjur OSS, Enterprise, Cloud)
```

```python
from conjur_api.providers.authn_authentication_strategy import AuthnAuthenticationStrategy
```

* OIDC authentication (Supported on Conjur OSS, Enterprise)
##### OIDC authenticator setup on [Conjur](https://docs.cyberark.com/AAM-DAP/13.0/en/Content/OIDC/OIDC.htm?tocpath=Integrations%7COpenID%20Connect%20(OIDC)%20Authenticator%7C_____0) 
```python
from conjur_api.providers.oidc_authentication_strategy import OidcAuthenticationStrategy
```

* LDAP authentication (Supported on Conjur OSS, Enterprise)
##### LADP authenticator setup on [Conjur](https://docs.cyberark.com/AAM-DAP/13.0/en/Content/Integrations/ldap/ldap_authenticator.html?tocpath=Integrations%7CLDAP%20Authentication%7C_____0)
```python
from conjur_api.providers.ldap_authentication_strategy import LdapAuthenticationStrategy
```

#### Define connection parameters

> In order to login to conjur you need to have 5 parameters known from advance.

Reviewer: ```suggestion
In order to login to conjur you need to have 5 parameters known in advance.
```
